VERSION_APPLIANCE

```diff
@@ -1 +1 @@
-5.0
+PR-57
```

Comment:
will need to be reverted later

Reply:
After merging this PR and being able to build appliance from master.